Doc/library/argparse.rst

```diff
@@ -641,6 +646,39 @@ the help options::
      +h, ++help  show this help message and exit
 
 
+add_manpage
+^^^^^^^^^^^
+
+By default, ArgumentParser objects will not add an option which display the
+parser's man page message. You should enable is when creating ArgumentParser
+obejcts. For example, consider a file named ``myprogram.py`` containing the
+following code::
+
+   import argparse
+   parser = argparse.ArgumentParser(add_manpage=True)
+   parser.add_argument('--foo', help='foo help')
+   args = parser.parse_args()
+
+If ``--manpage`` is supplied at the command line, the ArgumentParser
+man page will be open:
+
+.. code-block:: shell-session
+
+   $ python myprogram.py --manpage
+    tests.py(1)       General Commands Manual        tests.py(1)
+
+    NAME
+           tests.py
+
+    SYNOPSIS
+           tests.py [-h] [--manpage] [--foo FOO]
+
+    OPTIONS
+       optional arguments
+           -h, --help
+                  show this help message and exit
```

Comment:
If this the realy output?  I would expect the command to print out the man page markup, so that it can be saved as a `program.1` file and distributed by downstream packagers.